packages/fxa-auth-server/test/remote/account_create_tests.js

```diff
@@ -186,6 +199,90 @@ describe('#integration - remote account create', function () {
     );
   });
 
+  it('stubs account and finishes setup', async () => {
```

Comment:
@chenba Just calling this out since the `/account/stub` and `/account/finish_setup` weren't previously being integration tested.